### ISSUE TITLE
Enhancement Proposal: Keyboard shortcut helpers for cross-platform portability

### ISSUE THREAD

**Issue by pushfoo** [pyglet/pyglet#933]
#### tl;dr of problem

GUI keyboard shortcuts risk breaking when projects are moved between platforms:

1. There are cross-platform conventions for the letters of common keyboard shortcuts (`A` for select all, `C` for copy, `V` for paste, etc). 
2. Mac prefers the `Cmd` [modifier key](https://pyglet.readthedocs.io/en/latest/modules/window_key.html#key-constants) while everything else prefers `Ctrl`

Here's an example from #931:
```python
# Doesn't work on Mac since it uses MOD_CMD
if symbol == pyglet.window.key.A and modifiers & pyglet.window.key.MOD_CTRL:
    self._caret.select_all()
 ```

Addressing this seems to be in-scope for pyglet since it's an input abstraction rather than a game framework feature.

#### Potential Approaches

1. Add platform-dependent `MOD_PLATFORM_*` imports to the `pyglet.key` module
2. Add abstractions to the `pyglet.input` module
3. Extend `KeyStateHandler`

**Comment by pushfoo** [comment]
For #931, we may already have relevant helpers in the form of `key.MOTION_*`, `key.UNDO`, and other helpers. I'll investigate this further in a bit.

**Comment by pushfoo** [comment]
Nvm, this is already covered by `MOD_ACCEL`.
